docs/how-tos/create-a-new-event.rst

```diff
@@ -231,20 +231,20 @@ Here is how the integration could look like:
         )
 
 - Ensure that the event is triggered consistently and only when the event should be triggered. Avoid triggering the event multiple times for the same event unless necessary, e.g., when there is no other way to ensure that the event is triggered consistently.
-- Try placing the event after the triggering logic completes successfully to ensure that the event is triggered only when it's needed. This will help ensure that the event is triggered only for factual events if the triggering logic fails, the event should not be triggered.
+- Try placing the event after the triggering logic completes successfully to ensure that it is triggered only when needed.  This will help ensure that the event is triggered only for factual events. If the triggering logic fails, the event should not be triggered.
 
 Step 7: Test the Event
-~~~~~~~~~~~~~~~~~~~~~~~
+========================
 
-You should test the event to ensure it triggers consistently and that its payload contains the necessary information. Add unit tests in the service that triggers the event. The main goal is to verify that the event triggers as needed, consumers can react to it, and it carries the expected information.
+You should test the event to ensure it triggers consistently and that its payload contains the necessary information. Add unit tests to the service that trigger the event. The main goal is to verify that the event triggers as needed, that consumers can react to it, and it carries the expected information.
 
 To ensure that our example is tested thoroughly, we should:
 
-- Add unit tests to the ``enroll`` method to ensure that the event is triggered when a user enrolls in a course. This means testing the event is triggered when the enrollment process completes successfully.
+- Add unit tests to the ``enroll`` method to ensure that the event is triggered when a user enrolls in a course. This means testing the event is triggered when the enrollment process is completed successfully.
```

Comment:
```suggestion
- Add unit tests to the ``enroll`` method to ensure that the event is triggered when a user enrolls in a course. This means testing the event is triggered when the enrollment process is completes successfully.
```

"completes successfully" was fine, and is better than "is completed successfully". If you want to reword this the proper English would be "is successfully completed".